dodola/cli.py

```diff
@@ -191,3 +191,18 @@ def regrid(x, out, method, domain_file, weightspath):
         domain_file=domain_file,
         weights_path=weightspath,
     )
+
+
+@dodola_cli.command(help="Correct wet day frequency in a dataset")
+@click.argument("x", required=True)
+@click.option("--out", "-o", required=True)
+@click.option(
+    "--process",
+    "-p",
+    required=True,
+    type=click.Choice(["pre-process", "post-process"], case_sensitive=False),
```

Comment:
Minor suggestion to shorten things because you already have "--process" as part of the input.

```suggestion
    type=click.Choice(["pre", "post"], case_sensitive=False),
```

Reply:
Great idea, thanks!

Reply:
@dgergel Thanks but your other functions in core, services, and tests are still expecting (and documented for) `"pre-process"` and `"post-process"`. I think you'd need to switch those over for this to work.